encoding/kzg/prover/parametrized_prover.go

```diff
@@ -225,6 +225,7 @@ func (p *ParametrizedProver) proofWorker(
 				points: nil,
 				err:    err,
 			}
+			continue
```

Comment:
How about just moving the for loop below inside this code block?

Reply:
I used a else statement